src/less/tooltip/ds6/tooltip.less

```diff
@@ -32,6 +32,12 @@
     color: @color-black;
 }
 
+button.infotip__host.icon-btn {
+    height: 16px;
```

Comment:
The height and width already exist in the ds4 less file.  We should move them to the infotip base less file.

Reply:
I saw that but i split it because i am assuming heights and such will change for ds6.5 icons
If it wont then I'll combine

Reply:
If so, we can split them when we get new infotip designs. But if they don't change we'll probably forget about it.  So I'd say it would be better to fix it now, and split later if needed.